src/cascadia/TerminalApp/App.cpp

```diff
@@ -77,22 +77,7 @@ namespace winrt::TerminalApp::implementation
     /// <param name="e">Details about the launch request and process.</param>
     void App::OnLaunched(const LaunchActivatedEventArgs& /*e*/)

```

Comment:
Not necessarily as a part of this PR, but can we just outright remove this function and all the other pure UWP stuff that's left over?

Reply:
Probably. It's Real Dead.